ui/src/app/shared/model/error.model.ts

```diff
@@ -0,0 +1,91 @@
+/**
+ * Generic App Error
+ */
+export class AppError {
+
+  /**
+   * Message
+   */
+  message: string;
+
+  /**
+   * Constructor
+   * @param {string} message
+   */
+  constructor(message: string) {
+    this.message = message;
+  }
+
+  /**
+   * ToString
+   *
+   * @returns {string}
+   */
+  toString() {
```

Comment:
I would like to rename that to `getMessage()` - that way it feels a bit more similar to Java :-) Since coming from that world `toString()` may not imply a end-user-destined message but may contain other debug-info.